hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/AbstractBackendStore.java

```diff
@@ -25,9 +25,11 @@
 public abstract class AbstractBackendStore<Session extends BackendSession>
                 implements BackendStore {
 
+    private final SystemSchemaStore systemSchemaStore;
```

Comment:
TODO: move SystemSchemaStore into backend like MetaStore

Reply:
this TODO  annotation it first